### DISCUSSION TITLE
Import CSV is Clobbering the Start Time, End Time and Day of the Week

### DISCUSSION THREAD

**Discussion by MinnesotaSwanny** [code4recovery/12-step-meeting-list#1019] [Q&A] [answered]
Using the import CSV in the latest version of the plugin (3.14.9) is now clobbering the Start Time, End Time and the Day of the Week columns.  Help!
![7627D952-7C20-44BD-93E5-D31B1A7509F5](https://user-images.githubusercontent.com/103621510/209902605-6b574e4d-bbdf-4894-b16e-0a8a3b2c071a.png)

**Comment by joshreisner** [comment, marked as answer]
Again I'm transferring this issue out of the spec repository. The spec repository deals with our JSON specification, not the 12 Step Meeting List WordPress plugin. Going forward, if you have a question, please create a new discussion at https://github.com/code4recovery/12-step-meeting-list/. Thanks

I don't think the issue you're having is related to the recent plugin update. You reported in #1016 that you had uploaded a CSV and some meetings were showing up as appointment. I suggested that there might be something amiss with the Day and/or the Time column. Could we see that upload? In this screenshot it looks like you are now re-downloading the CSV, with the time information already dropped.

**Reply by MinnesotaSwanny** [reply]
You can close this out.  I found my problem.  I use a full .XLSX spreadsheet to maintain our SCI meeting list, which I then convert to a .CSV for import into the plug-in.  Well, last night I was not paying attention and I used "UTF-8" encoding which I believe caused my issues.  I am all good now.  Thanks!
